Doc/library/functions.rst

```diff
@@ -680,6 +680,15 @@ are always available.  They are listed here in alphabetical order.
    If the :mod:`readline` module was loaded, then :func:`input` will use it
    to provide elaborate line editing and history features.
 
+   .. versionchanged:: 3.7
+      If the input prompt string has a NULL character, then it will raise
```

Comment:
NULL means a null pointer in C. Use "a null character" for consistency with other documentation.

But actually I think this change doesn't need special mentioning in the documentation. This is just a bugfix. A null character is prohibited in many other APIs.